src/raiden_libs/user_address.py

```diff
@@ -69,6 +69,7 @@ def __init__(
         self._log = None
         self._listener_id: Optional[UUID] = None
         self._capabilities_schema = CapabilitiesSchema()
+        self._first_seen_offline: Dict[Address, Any] = {}
```

Comment:
I think we could type this as
```
Dict[Address, datetime]
```
or
```
Dict[Address, Optional[datetime]]
```
no?

Reply:
✅